### ISSUE TITLE
Show images on IE

### ISSUE THREAD

**Issue by lazaronixon** [elabs/refile#92]
IE dont support images without mime/type, if I use format option on attachment_url it works but i want to provide gif and jpg formats, some way to do this?

**Comment by lazaronixon** [comment]
I think the problem is that IE doesn't show images with "application/octet-stream" content type

**Comment by lazaronixon** [comment]
Sorry I didn't see option to save content_type, so I created a helper something like:
```ruby
ext = 'jpg' if ['image/jpeg','image/png', nil].include? post.image_content_type
ext = 'gif' if 'image/gif' == post.image_content_type

attachment_image_tag(post, :image, :fill, 300, 300, format: ext)
```
It is working ok.. But gif is shown without animation, any idea why?

**Comment by mdesantis** [comment]
> But gif is shown without animation, any idea why?

That's because you are processing it (`:fill, 300, 300`), and fill processing implementation makes gifs loosing animation; try without processing, you should see the animated GIF

**Comment by lazaronixon** [comment]
It was fixed.. About GIF #104

**Comment by mdesantis** [comment]
Nice :)

**Comment by jnicklas** [comment]
Seems like we were still not setting the content type properly. I just pushed a fix and released it as 0.5.1. This should now be working as intended :)
